uint16 camera
int8 CurrentShelf
---
bool isEmpty